Snail/Observable.swift

```diff
@@ -3,22 +3,25 @@
 public class Observable<T> : ObservableType {
     public typealias E = T
     private var isStopped: Int32 = 0
-    private var eventHandlers: [(Event<E>) -> Void] = []
+    var eventHandlers: [(Event<E>) -> Void] = []
```

Comment:
I don't think we want these to be non-private do we? maybe `internal` (or is that the default)?